conf/st2.package.conf

```diff
@@ -77,3 +77,6 @@ url = amqp://guest:guest@127.0.0.1:5672/
 
 [ssh_runner]
 remote_dir = /tmp
+
+[workflow_engine]
```

Comment:
`rulesengine` vs `workflow_engine` inconsistency makes me :cry: 

I think that's what @lakshmi-kannan mentioned a few days ago.

Reply:
Same here.

There are similar examples in other places. I personally prefer `_`.